apps/SageAccountWeb/package.json

```diff
@@ -13,6 +13,7 @@
     "@mui/utils": "^5.15.13",
     "@react-hookz/web": "^23.1.0",
     "@tanstack/react-query": "5.22.2",
+    "katex": "^0.16.10",
```

Comment:
Bump KaTeX (resolving some dependabot alerts). In this project, it was moved from devDependencies to dependencies.